### PR TITLE
Added "Server..." tutorial

### PR DESCRIPTION
**Pull request by emile-00**
## Info
- **Description**: 
Added the "Server performance anomaly detection" to the tutorial section. 

- **Notes**: 
[ Any additional information? ]

## Before merging
  - [x] (For version-specific PR) I have selected the corresponding software version in **Milestone** and linked the related doc issue to this PR in **Development**.
  - [x] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`bernscode`, `CharlieSYH`, `emile-00`, & `hengm3467`). 
  - [x] I have checked the doc site preview and the updated parts look all good. <details><summary>How?</summary>image</details>
  - [x] (For version-specific PR) I have confirmed that this PR is for a released version. If the software version is not released, put it on hold.

## Published doc pages
  [ After merging this PR, edit the description to include the links to the updated doc pages here. For example, https://www.risingwave.dev/docs/latest/intro/. ]

### CODE REVIEW

**Comment by hengm3467** [comment]
@emile-00  Please help update the title of the other tutorial from "Perform real-time ad performance analysis" to "Real-time ad performance analysis", so that it's consistent with the new tutorial

File: docs/tutorials/server-performance-anomaly-detection.md [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+
+## Step 3: Define materialized views and query the results
+
+In this tutorial, we will create a few different materialized views. The first view, `high_util_tcp_metrics`, will include average values for all metrics of every device every 3 minutes. The other three views will be derived from the first view, each containing a trigger time and different metric values.
+
+### Set up materialized view for highly utilized TCP metrics
+
+First, we will create the materialized view that contains all relevant TCP values. We use the tumble function to map all events into 1-minute windows and calculate the average metric value for each device within each time window. Next, the average TCP and NIC metrics are calculated separately before joining on device names and time windows. We will keep the records measuring the volume of bytes transferred by the interface and where the average utilization is greater than or equal to 50.
+
+Please refer to this [guide](https://www.risingwave.dev/docs/latest/sql-function-time-window/) for an explanation of the tumble function and aggregations.
```
Comment (CharlieSYH) [inline docs/tutorials/server-performance-anomaly-detection.md]: We should use the file path instead as the domain name 'www.risingwave.dev' could change in the future.
Revision: `[guide](/sql/functions-operators/sql-function-time-window.md)`

**Review by CharlieSYH — CHANGES_REQUESTED** [review]

**Review by hengm3467 — APPROVED** [review]
Looks good!

**Review by CharlieSYH — APPROVED** [review]
Looks great!

**Pull request merged**